_components/table-2.html

```diff
@@ -12,20 +12,20 @@
             <!-- Inputs -->
             <div class="flex flex-grow p-1 space-x-1">
                 <div class="relative flex">
```

Comment:
I think this element could be how you define whether it has an icon. It doesn't have this wrapper unless it has an icon. So this wrapper could be `.htk-input-with-icon` and the icon could be `.htk-input-with-icon i` (instead of making a new class, we're always using `<i>`), and the input could just be `.htk-text-input` or `.htk-select-input` etc.

Reply:
Looking closer at this, it appears all select boxes have this wrapper due to them having the chevron icon needed. I think there's still something to this approach, but keep that in mind. I would have to think into how I would do this, but I'd like to see what your approach is.